src/charts/BalanceHistory.js

```diff
@@ -2,20 +2,43 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Chart from 'react-apexcharts';
 import moment from 'moment';
+import { Paper } from '@material-ui/core';
 
 const BalanceHistory = ({ transactions }) => {
   let result = [];
 
   transactions.reduce((previous, current) => {
     const balance = previous + current.amount;
     result = result.concat({
-      x: moment(current.date).format('dddd, MMMM Do YYYY'),
+      x: moment(current.date),
       y: balance
     });
     return balance;
   }, 0);
 
-  const options = {};
+  const options = {
+    xaxis: {
+      type: 'datetime',
+      labels: { show: false }
+    },
+    chart: {
+      zoom: {
+        enabled: true,
+        type: 'x',
+        zoomedArea: {
+          fill: {
+            color: '#90CAF9',
+            opacity: 0.4
+          },
+          stroke: {
+            color: '#0D47A1',
```

Comment:
would be nice to use the theme primary color palette too.. But no need to do anything about this :)